core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -417,5 +417,33 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
 
       }
     }
+
+    "consume starting from a specific timestamp when provided" in {
+      assertAllStagesStopped {
+        val topic = createTopic(1)
+        val group = createGroup(1)
+
+        givenInitializedTopic(topic)
+
+        Await.result(produce(topic, 1 to 2), remainingOrDefault)
+
+        val seekToTimestamp = System.currentTimeMillis()
+        val toBeConsumed = 2 to 4
```

Comment:
since `to` is inclusive, might `3 to 4` make more sense?

Reply:
I agree, fixed.